scene/main/viewport.h

```diff
@@ -672,6 +672,8 @@ class Viewport : public Node {
 	void set_canvas_cull_mask_bit(uint32_t p_layer, bool p_enable);
 	bool get_canvas_cull_mask_bit(uint32_t p_layer) const;
 
+	bool is_visible_subviewport() const;
```

Comment:
Would it make sense to add a `#ifdef TOOLS_ENABLED` check, because that function is used only in the editor?